pycalphad/core/equilibrium.py

```diff
@@ -239,22 +247,25 @@ def equilibrium(dbf, comps, phases, conditions, output=None, model=None,
     if verbose:
         print('[done]', end='\n')
 
-    # 'calculate' accepts conditions through its keyword arguments
-    grid_opts = calc_opts.copy()
-    statevar_strings = [str(x) for x in state_variables]
-    grid_opts.update({key: value for key, value in str_conds.items() if key in statevar_strings})
-    if 'pdens' not in grid_opts:
-        grid_opts['pdens'] = 500
-    grid = calculate(dbf, comps, active_phases, model=models, fake_points=True,
-                     callables=callables, output='GM', parameters=parameters, 
-                     to_xarray=False, **grid_opts)
+    if global_min:
```

Comment:
I think this will fail if `(not global_min) and (not user_starting_point)` is True, since `starting_point()` will have no grid.